specific_use_cases/DFU_work_in_progress/station_8760s.ipynb

```diff
@@ -0,0 +1,627 @@
+    "    stat_median.hvplot(groupby='station',kind='scatter',c='k') * \\\n",
+    "    median_year_by_station.hvplot(groupby='station',kind='line',lw=0.5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7d536520-d38a-47c2-8e0e-209dd1fd0f9b",
+   "metadata": {},
+   "source": [
+    "#### Typical year heatmap\n",
+    "Turning back to that closest-to-the-median year, with all of its variability, let's view it another way.\n",
+    "\n",
+    "You may want to rearrange the timeseries to view as a heatmap over day-of-year vs hour-of-day. There may be other ways to do this, but here we will add dimensions for day and hour, and then convert into a pandas dataframe, which we can more easily pivot."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a25a02b0-b5ab-403d-9351-999d6d5601cf",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from copy import deepcopy\n",
+    "median_year_array = deepcopy(median_year)\n",
+    "median_year_array['time'] = pd.date_range('19810101',periods=8760,freq='H')\n",
+    "median_year_array['hour'] = median_year_array['time.hour']\n",
+    "median_year_array['dayofyear'] = median_year_array['time.dayofyear']\n",
+    "\n",
+    "median_year_df = median_year_array.to_pandas().drop(['scenario','simulation'],axis=1) \n",
+    "median_year_df = median_year_df.pivot(index=\"dayofyear\", columns=\"hour\")\n",
+    "median_year_df.columns.names = ['station', 'hour']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5aa3971e-800b-4865-92bc-d5b12101e312",
+   "metadata": {},
+   "source": [
+    "Then put it back into an xarray datastructure again for better plotting of heatmaps, and examine the new data structure."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "863d6e21-37f1-4e5b-9d60-933c20a33eb1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "median_year_array = median_year_df.unstack().to_xarray()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0588f7c4-7f86-4500-a251-b627e7313e49",
+   "metadata": {},
+   "source": [
+    "With just a little attention to formatting in local time."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b5b1de4c-c923-4f37-b1a6-c647ca5c00d5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def to_PST(hour):\n",
+    "    #take hour 0-23, assumed UTC\n",
+    "    #return formatted Pacific Standard Time string\n",
+    "    return pd.to_datetime(hour*60, origin=pd.Timestamp('1981-01-01'),unit='m', utc=True).tz_convert('US/Pacific').strftime('%r')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "aab21ce6-148a-4408-9d41-2f3e7727a136",
+   "metadata": {},
+   "source": [
+    "Finally, the heatmap:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "65ea17fd-bba4-4a31-95a0-014065b4b7f3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "median_year_array.hvplot(\n",
```

Comment:
A nice to have - label the color ramp with the variable & units represented

Reply:
Done, although it's not done in the ideal manner in which I'd do it in a climakitae function, since I'm checking app.selections, and that could change in the meanwhile if people don't execute cells in order.